samples/kvsWebRTCClientMaster.c

```diff
@@ -50,6 +50,21 @@ INT32 main(INT32 argc, CHAR* argv[])
     pSampleConfiguration->audioCodec = audioCodec;
     pSampleConfiguration->videoCodec = videoCodec;
 
+    // Configure the RTP rolling buffer sizes for the set of sample frames
+    // to be smaller than the default settings (plus some extra for padding).
+    if (pSampleConfiguration->videoCodec == RTC_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE) {
+        pSampleConfiguration->videoRollingBufferDurationSec = 3;
+        pSampleConfiguration->videoRollingBufferBitratebps = 1.4 * 1024 * 1024;
```

Comment:
Note: In the other samples this is not set, meaning default values are used. Since CALLOC is used for the sample configuration, these values are 0 everywhere else. Calling `configureTransceiverRollingBuffer` with 0's (or also not calling it at all) will use the default values.